README.md

```diff
@@ -1,25 +1,120 @@
+  * Relies on [osmconvert](https://wiki.openstreetmap.org/wiki/Osmconvert) tool to extract the region
+of interest from an OSM map.
+* the second one sets up a lightweight tileserver-gl using the data generated above;
 
-## Usage
+### Shapefile
+
+### Styles
+
+### Build command
+
+As for any Docker container that you want to build from a Dockerfile, go into the directory
+containing the Dockerfile and run the following command:
 
 ```
 $ docker build . -t indico-maps
-$ docker run -p 8080:8080 -it indico-maps
 ```
 
-Then, just connect to https://yourhost:8080.
+''Note: `indico-maps` will be the tag of (name referring to) the container. You can use any
+name you prefer: nothing in the configuration refers to this name.`
+
+## Running the tile server
+
+### tileserver configuration
+
+The `tileserver` configuration file is located in `tileserver/config.json` in the repository.
+It contains 2 dictionaries which have normally one key each. The key must be the same in both
+dictionaries and will be your site name (used in the tile server URL configured in Indico).
+
+The 2 dictionaries are:
+* `data`: it contains an entry `mbtiles` which is the name of the file that contains the map
+tiles. It must match the name of the file produced by `tilemaker`.
+* `styles`: this dictionary has 2 entries:
+  * `styles`: must reflect you style configuration, i.e. the `config.json` path relative to the
+  `styles` directory.
+  * `tilejson/bounds`: an array of GPS coordinates of your site OSM map (processed by tilemaker)
+  for the south-west and the north-east corners with `latitude` the `longitude` for each one.
+
+### Starting the tileserver
+
+The typical command to run the tileserver container built previously is:
+
+```
+$  docker run -dit --restart unless-stopped -p 8080:8080 indico-maps
+```
+
+''Note: if you use a container tag different from `indico-maps` when you built the container,
+use it when starting the container.`
+
+The command above runs the container as a daemon (`-d`) and ensures it is restarted when Docker
+starts (for example after a reboot). If you want to see the tileserver output (logs), you must
+use the command `docker attach` with the container ID returned by the `run` command. Type CTRL/C
+to exit `docker attach`.
+
+### Testing the tile server
+
+To test your tile server, the easiest is to use a browser and connect to `https://yourhost:8080`.
+
+### Using from Indico
+
+To use the tile server in Indico room booking, goto the room booking administration panel and
+in the tileserver URL field, enter something similar to:
+
+```
+http://your.tileserver.dom.ain/styles/$sitename/{z}/{x}/{y}.png
+```
+
+`$sitename` must be replaced by the key you used in the dictionaries of your tileserver 
+configuration.
+
+### Accessing the server with https
+
+`tileserver` doesn't support `https` natively. If you want to access the tileserver through https,
```

Comment:
Can we be more explicit about using HTTPS here? Mixed content likely won't work well (and if it does it's at least very ugly). Maybe something like "Since your production Indico instance is running on HTTPS, you need to use HTTPS for the tile server as well."